pantab/__init__.py

```diff
@@ -109,6 +113,6 @@ def _get_hapi_function(name, sig):
     ),
     _get_hapi_function(
         "hyper_rowset_chunk_field_values",
-        "struct hyper_error_t *(*)(struct hyper_rowset_chunk_t *, size_t *, size_t *, uint8_t * * *, size_t * *, int8_t * *)",
```

Comment:
This is different than what is in tableauhyperapi.h - is that intentional?

Reply:
Yea this was in reference to the `const` qualifiers - didn't realize cffi dropped those